extension/intents/tabs/tabs.toml

```diff
@@ -377,7 +377,49 @@ match = """
 
 [[tabs.findOnPage.example]]
 phrase = "Find query on this page"
-test = true
 
 [[tabs.findOnPage.example]]
-phrase = "Find spaghetti on  page"
\ No newline at end of file
+phrase = "Find spaghetti on  page"
+
+[[tabs.selectAllTabs]]
+description = "Select all tabs"
+match = """
+    (select | highlight) (all |) (the | this ) (tab{s} | taps | page )
+"""
+[[tabs.selectAllTabs.example]]
+phrase = "Select all the tabs"
+
+[[tabs.selectAllTabs.example]]
+phrase = "Select all tabs"
+
+[[tabs.highlightSelectedTabs]]
+description = "Select tabs to the left/right"
+match = """
+    (select | highlight) all (tab{s} | page) (to | in | by |) (the |) right [dir = right] [allWindows = false]
+    (select | highlight) all (tab{s} | page) (to | in | by |) (the |) left  [dir = left] [allWindows = false]
```

Comment:
I'd use `(all |)` to make that optional, and `page{s}` to match both page and pages.